x/oracle/keeper/migrations.go

```diff
@@ -0,0 +1,23 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// Migrator is a struct for handling in-place store migrations.
+type Migrator struct {
+	keeper *Keeper
+}
+
+// NewMigrator creates a Migrator.
+func NewMigrator(keeper *Keeper) Migrator {
+	return Migrator{keeper: keeper}
+}
+
+// Migrate1to2 migrates from version 1 to 2.
+func (m Migrator) Migrate1to2(ctx sdk.Context) error {
+	m.keeper.SetStampPeriod(ctx, 1)
+	m.keeper.SetPrunePeriod(ctx, 1)
+	m.keeper.SetMedianPeriod(ctx, 1)
```

Comment:
Hmm feels weird to have these set to 1, but that's a better default than 0 since we'd have to remove validation

Reply:
yes, we just need to put some number there anyway.